packages/service-client/odsp-client/README.md

```diff
@@ -0,0 +1,101 @@
+## @fluid-experimental/odsp-client
+
+The odsp-client package provides a simple and powerful way to consume collaborative Fluid data with the ODSP service.
```

Comment:
consume data with service? Does not sound right to me. Maybe build collaborative Fluid applications with ODSP as a storage mechanism?